src/App/WP/Tamara_Checkout_WP_Plugin.php

```diff
@@ -101,6 +109,26 @@ public function get_tamara_gateway_service(): Tamara_WC_Payment_Gateway {
 		return wp_app( Tamara_WC_Payment_Gateway::class );
 	}
 
+	public function get_tamara_gateway_single_checkout_service() {
+		return wp_app( Single_Checkout_WC_Payment_Gateway::class );
+	}
+
+	public function get_tamara_gateway_pay_next_month_service() {
+		return wp_app( Pay_Next_Month_WC_Payment_Gateway::class );
+	}
+
+	public function get_tamara_gateway_pay_now_service() {
+		return wp_app( Pay_Now_WC_Payment_Gateway::class );
+	}
+
+	public function get_tamara_gateway_pay_in_x_service( $instalment ) {
+		return wp_app( 'Tamara_Checkout\App\WP\Payment_Gateways\Pay_In_' . $instalment . '_WC_Payment_Gateway' );
+	}
+
+	public function get_customer_phone_number() {
```

Comment:
What is `$this->customer_phone_number` for?

Reply:
this is used for adjusting payment methods anytime a customer change their phone number on checkout. We will need to call an ajax to handle it through js later.